src/Miso/WebSocket.hs

```diff
@@ -46,7 +43,7 @@ import           Miso.Runtime
 import           Miso.String (MisoString)
 import           Miso.FFI (Blob(..), ArrayBuffer(..))
 -----------------------------------------------------------------------------
--- | <https://developer.mozilla.org/en-US/docs/Web/API/WebSocket/connect>
+-- | <https://developer.mozilla.org/en-US/docs/Web/API/WebSocket>
```

Comment:
There's nothing at the original url. The general intro to the WebSocket api is the best I could find (because connection is triggered by calling the WebSocket js constructor..)